### DISCUSSION TITLE
Error in mesh export process

### DISCUSSION THREAD

**Discussion by gtrewq** [nextfoam/baram#106] [Q&A]
Hello,

I am beginner of baram, and started to use NextFOAM-2407 on balam 24.3.1. I would like to run simulation case with fan and porous jump boundary conditions. Therefore, a mesh containing interface boundaries (conformal) were created, but the process did not proceed from the state shown in the attached image when the mesh was exported. The following error messages were displayed in the console

```
[2024-08-07 16:46:50,470][root] ==> Uncaught exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/qasync/__init__.py", line 783, in _error_handler
    task.result()
  File "/opt/baram/baramMesh/view/export/export_page.py", line 145, in _export
    await redistributionTask.reconstruct()
  File "/opt/baram/baramMesh/openfoam/redistribution_task.py", line 62, in reconstruct
    raise RuntimeError(self.tr('Mesh Reconstruction failed.'))
RuntimeError: Mesh Reconstruction failed.
```

I would appreciate you if you  provide a solution to this error.

Best regards,
Junichi Fukui
![error_reconstruct_mesh](https://github.com/user-attachments/assets/fa030ae4-6e71-48e5-ad08-a2013e757569)

**Comment by jiban** [comment]
Dear Junichi,
I could not find a clue for now.
Could you give me the case you tried?
Please compress the entire project directory as soon as you encounter the problem.
Thank you.

Best regards,
Jake

**Comment by gtrewq** [comment]
Dear Jake,

Thank you for the prompt reply.

The compressed case file was uploaded to the [cloud server](https://glm36501-my.sharepoint.com/:u:/g/personal/junichi_fukui_glm_jp/ET6j3wZXXBdNqyXaI8dKXsgBoegsi9Zn-8XOnthycm4yKQ?email=jakejiwon%40gmail.com&e=heSnqG). Could you download and check the case?

Best regards,
Junichi

**Comment by jiban** [comment]
Hello Junichi,
I've received the case, yet I could not handle the case because it is too big for me.
Mr.Kim, who is one of my colleagues, will look into it and get back to you.
Thank you.

**Comment by kdg5424** [comment]
Hi Junichi. This is Donggyu from NEXTFOAM.
I'm checking your case. I'll let you know what is the reason as soon as possible.
I'm really sorry for inconvenience.

Regards

**Comment by gocfd** [comment]
Hello Junichi,

I faced similar issue when exporting mesh file. Apparently this happens only when using several cores for meshing in Parallel Evironment. After changing to 1 core, mesh was successfully exported. I hope this helps. 

Best regards

**Comment by kdg5424** [comment]
Hello all guys.
This problem has been solved in BARAM V24.3.3
So if you use BARAM V24.3.3, there would be no problem.

Thanks

Donggyu kim
